spec/helpers/schema.rb

```diff
@@ -9,6 +9,15 @@
 
   create_table :posts do |t|
     t.references :user, foreign_key: true
+    t.string :type
+    t.references :level, foreign_key: true
+
+    t.timestamps null: false
+  end
+
+  create_table :levels do |t|
+    t.references :post, foreign_key: true
```

Comment:
Looks like this line isn't necessary

Reply:
@DmitryTsepelev should I create separate PR or some is going to remove that as a part of the PR?

Reply:
@konalegi please make a separate PR since this one is already merged

Reply:
PR to cleanup https://github.com/DmitryTsepelev/ar_lazy_preload/pull/41